### ISSUE TITLE
unable to get data from the database

### ISSUE THREAD

**Issue by shubhsingh594** [joebowen/model_completion_dashboard#9]
using " pip install pyopenworm " we are unable to get data from the database. 

<img width="1371" alt="screen shot 2017-02-15 at 5 37 41 pm" src="https://cloud.githubusercontent.com/assets/15982349/23163285/c50e5832-f857-11e6-8a85-76b082525ae8.png">

however when we clone the PyOpenWorm repository and then run the dashboard, we start getting data from the database. 

<img width="1245" alt="screen shot 2017-02-21 at 5 06 43 pm" src="https://cloud.githubusercontent.com/assets/15982349/23163377/38ebd81a-f858-11e6-9fea-dae5cd786a0e.png">

https://github.com/openworm/PyOpenWorm/issues/301

as suggested by this link this is because post_install.py is not executed when installing from pip.

**Comment by shubhsingh594** [comment]
can i work on this issue ?

**Comment by jamiebbowen** [comment]
Of course!  And if you need any permissions or pull requests accepted, please let me know.  :-)
